spa/src/components/dashboard/Dashboard.js

```diff
@@ -36,7 +36,7 @@ function Dashboard() {
     ? DONATIONS_SLUG
     : 'not-found';
 
-  const isEditPage = useLocation().pathname.includes('/dashboard/edit');
+  const isEditPage = useLocation().pathname.includes('/edit');
```

Comment:
Would be better to use the constant so that if it changes this verification is always updated
```suggestion
  const isEditPage = useLocation().pathname.includes(EDITOR_ROUTE);
```

Reply:
done